Automattic/sensei
# Add CC/BCC fields for Sensei emails

Resolves #7001 

In this PR:

- I moved adding our fields to Sensei_Settings class, to have only one place for all fields.
- I used a custom type for the field ("email_list") and a corresponding validation methods.
- These methods don't remove duplicated emails. Wasn't sure if we need it. Also it will affect validation in its current state.


## Proposed Changes
* Add CC/BCC fields for Sensei emails.

## Testing Instructions


1. Go to Sensei LMS -> Settings -> Email -> Settings.
2. Try to set invalid emails for CC & BCC.
3. It is expected to see an error message for each field.
4. Set different emails for CC and BCC.
5. Create a course with lessons.
6. Take the course as a student and complete it.
7. Check your emails. It is expected to get an email in accounts you set for CC & BCC. The email that was sent to BCC shouldn't have that address in recipients.

## Deprecated Code


* `Sensei\Internal\Emails\Email_Settings_Tab::add_reply_to_setting` – adding of these settings moved to \Sensei_Settings.

## Pre-Merge Checklist
